tidb-cloud-in-public-preview.md

```diff
@@ -0,0 +1,49 @@
+---
+title: TiDB Cloud Now in Public Preview
+author: [PingCAP]
+date: 2021-05-10
+summary: We are excited to announce the public preview of TiDB Cloud, the fully-managed TiDB service by PingCAP. With TiDB Cloud, customers can now easily use TiDB on Amazon Web Services and Google Cloud to quickly build modern, mission critical applications.
+tags: ['Announcement', 'TiDB Cloud']
+categories: ['Product']
+image: /images/blog/tidb-cloud-in-public-preview.jpg
+press_release: true
+---
+
+![TiDB Cloud in Public Preview](media/tidb-cloud-in-public-preview.jpg)
+
+We are excited to announce the public preview of TiDB Cloud, the fully-managed TiDB service by PingCAP. With TiDB Cloud, customers can now easily use TiDB on Amazon Web Services and Google Cloud to quickly build modern, mission critical applications.
+
+## Get the full power of TiDB
+
+TiDB is the leading open-source, MySQL compatible, distributed NewSQL database in the industry that supports hybrid transactional and analytical processing (HTAP), which allows business to run real-time analytical queries. It also features horizontal scalability, strong consistency, and high availability.
+
+*   **More control over your scaling**: Scale your performance (TiDB) and storage (TiKV) nodes separately 
+*   **High availability and reliability**: Ensure business continuity for your mission critical applications with our ACID-compliant cluster deployment
+*   **Scalable Multi-Master: **No more bottlenecks - scale without restrictions with read and write capability on every node
```

Comment:
```suggestion
*   **Scalable Multi-Master:** No more bottlenecks - scale without restrictions with read and write capability on every node
```

Reply:
For BlackLivesMatter issues, we don't use expressions like "master-slave." Instead, we use "primary-secondary."